docs/topics/data_calculation.md

```diff
@@ -116,4 +116,34 @@ For those cases when the data values that want to be calculated are statistical
 the [Short Term Historic](../sth.md) can be used. This component stores historical information of Context Entities and can be used
 to retrieve statistical calculations over the stored data.
 
-// @GERMAN: SOME INFORMATION ABOUT HOW TO GET STATISTICS CALCULATED BY THE STH WOULD BE NEEDED
\ No newline at end of file
+The STH component is able to calculate statistics about the evolution in time of certain entity attributes. To do it, there are 2 ways to notify this evolution in time to the component:
+
+1. Via the [Cygnus](https://github.com/telefonicaid/fiware-cygnus) component and subscribing it to the Context Broker instance.
+2. Via the [STH](https://github.com/telefonicaid/fiware-sth-comet) component itself and directly subscribing it to the Context Broker instance.
+
+Once the STH receives the notifications of entity attribute value changes, it is able to calculate and provide information about:
+
+* Numeric attribute values:
+    * Mean
+    * Standard deviation
+    * Variance
+    * Maximum
+    * Minimum
+* Textual attribute values:
+    * Number of occurrences
+
+All this, for distinct resolutions or time frames, such as:
+
+* Months
+* Days
+* Hours
+* Minutes
+* Seconds
+
+Regarding the previous example, the STH is able to provide statistical information such as:
+
+1. Which has been the maximum `weight` (attribute) values of the `WasteTank8` (entity) last year with a resolution of months?, sending a GET request to the STH component such as the next one: `http://<sth-component>:<sth-port>/STH/v1/contextEntities/type/Entity/id/WasteTank8/attributes/weight?aggrMethod=max&aggrPeriod=month&dateFrom=2015-01-01T00:00:00&dateTo=2015-12-31T23:59:59`
```

Comment:
The result of writing the query in the same sentence is strange. Can you move it to a code block below?